composer.json

```diff
@@ -5,15 +5,12 @@
     "license": "BSD-3-Clause",
     "config": {
         "sort-packages": true,
-        "platform": {
-            "php": "7.4.99"
-        },
```

Comment:
Should be kept, but updated

Reply:
8.0.99 ?

Reply:
yup

This prevents incompatible dependency auto-upgrades

Reply:
@fezfez was the lockfile updated?

Reply:
> @fezfez was the lockfile updated?

yep

Reply:
> @fezfez it states:
> 
> ```
>  Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
> ```

done